dev/npx.ts

```diff
@@ -1,34 +1,58 @@
+const suggestions: Fig.Suggestion[] = [
+  {
+    name: "react-native",
+    icon: "https://reactnative.dev/img/pwa/manifest-icon-512.png",
+  },
+  {
+    name: "tailwindcss",
+    icon: "https://tailwindcss.com/favicon-32x32.png",
+  },
+  {
+    name: "next",
+    icon: "https://nextjs.org/static/favicon/favicon-16x16.png",
+  },
+  {
+    name: "gltfjsx",
+    icon: "https://raw.githubusercontent.com/pmndrs/branding/master/logo.svg",
+  },
+  {
+    name: "prisma",
+    icon:
+      "https://raw.githubusercontent.com/prisma/docs/main/src/images/favicon-16x16.png",
+  },
+];
 export const completionSpec: Fig.Spec = {
   name: "npx",
   description: "Execute binaries from npm packages",
-  subcommands: [
-    {
-      name: "react-native",
-      icon: "https://reactnative.dev/img/pwa/manifest-icon-512.png",
-      loadSpec: "react-native",
-    },
-    {
-      name: "tailwindcss",
-      icon: "https://tailwindcss.com/favicon-32x32.png",
-      loadSpec: "tailwindcss",
-    },
-    {
-      name: "next",
-      icon: "https://nextjs.org/static/favicon/favicon-16x16.png",
-      loadSpec: "next",
-    },
-    {
-      name: "gltfjsx",
-      icon: "https://raw.githubusercontent.com/pmndrs/branding/master/logo.svg",
-      loadSpec: "gltfjsx",
-    },
-    {
-      name: "prisma",
-      icon:
-        "https://raw.githubusercontent.com/prisma/docs/main/src/images/favicon-16x16.png",
-      loadSpec: "prisma",
-    },
-  ],
+
+  args: {
+    name: "command",
+    isCommand: true,
+    generators: {
+      script: `until [[ -d node_modules/ ]] || [[ $PWD = '/' ]]; do cd ..; done; ls -1 node_modules/.bin/`,
+      postProcess: function (out) {
+        out.split("\n");
+
+        const cli = [
+          "react-native",
+          "tailwindcss",
+          "next",
+          "gltfjsx",
+          "prisma",
+        ];
```

Comment:
We can avoid duplicating those specs with something like this:
```suggestion
        const cli = [...suggestions].reduce((acc, { name }) => [...acc, name], []);
```